backend/src/database/migrations/20230710111307-create-govsg-template-access-table.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('govsg_templates_access', {
+      template_id: {
+        type: Sequelize.INTEGER,
+        primaryKey: true,
+        onDelete: 'NO ACTION',
+        allowNull: false,
+        references: {
+          model: 'govsg_templates',
+          key: 'id'
+        }
+      },
+      user_id: {
+        type: Sequelize.INTEGER,
+        primaryKey: true,
+        onDelete: 'NO ACTION',
+        allowNull: false,
+        references: {
+          model: 'users',
+          key: 'id'
+        }
+      },
+      created_at: {
+        type: Sequelize.DataTypes.DATE,
+        allowNull: false,
+      },
+      updated_at: {
+        type: Sequelize.DataTypes.DATE,
+        allowNull: false,
+      },
+    })
```

Comment:
I think should add an index `unique(template_id, user_id)` here so can search faster, and also to enforce uniqueness

Reply:
if it's alr a composite primary key, then there's alr a uniqueness constraint right 🤔️